### PR TITLE
Discussion conclusion revs

### PR DESCRIPTION
**Pull request by cgreene**
I ended up bringing the two sections together into one, since neither was particularly long. The current results also incorporate more discussion of the findings. Take a look and let me know what you think.

### CODE REVIEW

File: content/06.conclusion.md [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
+We analyzed the full text of bioRxiv, comparing it with the open access portion of PMC, and found that the overall manner of writing is consistent with the biomedical literature.
+A benefit of analyzing bioRxiv text is that we can compare preprint and published versions to examine the influence of the publication process.
+Token-level analyses suggest that differences between corpora are driven by fields, while comparisons of preprints with their corresponding publication reveals differences in typesetting and supplementary materials.
+
+Previous analyses have focused on article metadata, including which papers are heavily downloaded and discussed [@doi:10.7554/eLife.45133] and by which communities discuss them [@doi:10.1371/journal.pbio.3000860].
+We found that some preprints have high similarity to published articles in the PMC open access corpus, and a detailed examination revealed many that were published but not previously annotated.
+Using the full text of documents to correct these annotations provides a fuller understanding of the extent to which preprints are published, resulting in a publication rate that is higher than has been estimated.
```
Comment (danich1) [inline content/06.conclusion.md:9]:
```suggestion
We found that some preprints are high similar to published articles within the PMC open access corpus, and a detailed examination revealed that many preprints were published but not previously annotated.
Using the full text of documents to correct these missing annotations provides a comprehensive understanding of the extent to which preprints are published and this correction resulted in a publication rate that is higher than previously estimated.
```

File: content/06.conclusion.md [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
+Importantly, this only accounts for papers that are published open access, so our analysis should be considered to raise the lower bound but the truly published fraction is likely to be higher.
+
+Our work presents a first step towards understanding the process by which peer review alters life sciences papers through an analysis of the full text of preprints and the corresponding publication.
```
Comment (danich1) [inline content/06.conclusion.md:12]:
```suggestion
Our work presents a first step towards understanding the process by which peer review alters life sciences papers through an analysis of the full text of preprints and their corresponding publication.

```

File: content/06.conclusion.md [hunk 3]
```diff
@@ -1,7 +1,20 @@
...
+Our broad-based examination suggests certain changes, but the scale of changes appears modest.
+We lay the groundwork for future work that aims to identify sentences and claims that are altered during review.
+The finding that document embeddings reveal manuscripts with similar outcomes is likely to be important to new tools that accelerate publishing, including those that automatically perform integrity checks and other critically important contributions.
```
Comment (danich1) [inline content/06.conclusion.md:15]:
```suggestion
Our finding that document embeddings reveal manuscripts with similar outcomes is likely to be important to new tools that accelerate publishing, including those that automatically perform integrity checks and other critically important contributions.
```

File: content/06.conclusion.md [hunk 4]
```diff
@@ -1,7 +1,20 @@
...
+
+We found that document embeddings captured many elements of how authors use language in preprints.
```
Comment (danich1) [inline content/06.conclusion.md:17]:
```suggestion
We also found that document embeddings captured many elements of how authors use language in preprints.

```

**Review by danich1 — APPROVED** [review]
LGTM!

**Comment by cgreene** [comment]
Hit the too many requests error:
> docker: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.

Since manubot built I'm pretty confident this will work

**Pull request merged**